README.md

```diff
@@ -7,51 +7,93 @@
-- MacOS currently does not have any packages. We are still working on setting up MacOS packaging.
+- [System Package (AMD64)](https://packages.polymc.org/latest/lin64-system/lin64-system.tar.zst) ([SHA256](https://packages.polymc.org/latest/lin64-system/lin64-system.tar.zst.sha256)) - a generic system package intended to be used as a base for making distro-specific packages
 
-## Development
-If you want to contribute to PolyMC you might find it useful to join [#development:polymc.org on Matrix](https://matrix.to/#/#development:polymc.org) or join [our Discord server](https://discord.gg/xq7fxrgtMP), which is bridged with the PolyMC Matrix rooms. Thank you!
+### <img src="https://www.vectorlogo.zone/logos/archlinux/archlinux-icon.svg" height="20"/> Arch Linux
 
-### Building
-If you want to build PolyMC yourself, check [BUILD.md](BUILD.md) for build instructions.
+There are several AUR packages available:  
+[![polymc](https://img.shields.io/aur/version/polymc?label=aur/polymc)](https://aur.archlinux.org/packages/polymc/)  
+[![polymc-bin](https://img.shields.io/aur/version/polymc-bin?label=aur/polymc-bin)](https://aur.archlinux.org/packages/polymc-bin/)  
+[![polymc-git](https://img.shields.io/aur/version/polymc-git?label=aur/polymc-git)](https://aur.archlinux.org/packages/polymc-git/)
 
-You can build the flatpak using [this source](https://github.com/flathub/org.polymc.PolyMC).
+```sh
+# stable source package
+yay -S polymc
+# stable binary package:
+yay -S polymc-bin
+# latest git package:
+yay -S polymc-git
+```
 
-### Code formatting
-Just follow the existing formatting.
+A prebuilt Arch package is available:  
+[Arch Linux (AMD64)](https://packages.polymc.org/latest/arch/polymc-bin-latest-1-x86_64.pkg.tar.zst) ([SHA256](https://packages.polymc.org/latest/arch/polymc-bin-latest-1-x86_64.pkg.tar.zst.sha256))
 
-In general, in order of importance:
-* Make sure your IDE is not messing up line endings or whitespace and avoid using linters.
-* Prefer readability over dogma.
-* Keep to the existing formatting.
-* Indent with 4 space unless it's in a submodule.
-* Keep lists (of arguments, parameters, initializers...) as lists, not paragraphs. It should either read from top to bottom, or left to right. Not both.
+```
+wget https://packages.polymc.org/latest/arch/polymc-bin-latest-1-x86_64.pkg.tar.zst
+sudo pacman -U polymc-bin-latest-1-x86_64.pkg.tar.zst
+```
 
-## Translations
-TODO
+### <img src="https://www.vectorlogo.zone/logos/debian/debian-icon.svg" height="20" /> Debian
 
-## Forking/Redistributing/Custom builds policy
-Do whatever you want, we don't care. Just follow the license. If you have any questions about this feel free to ask in an issue.
+A prebuilt deb package is available:  
+[Debian (AMD64)](https://packages.polymc.org/latest/deb/polymc-amd64.deb) ([SHA256](https://packages.polymc.org/latest/deb/polymc-amd64.deb.sha256))
```

Comment:
This will be removed. Link the package in MPR. And is there an MPR package manager?

Reply:
Sure. There is tap which is the main helper for it